assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,25 @@
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.c-logged-in-as__icon {
+  height: 32px;
+  width: 32px;
+}
+
+.c-logged-in-as__text {
+  font-weight: 400;
+  font-size: 14px;
+  color: #6e7175;
+
+  p {
+    line-height: 21px;
```

Comment:
question: can unitless values be used instead here?
[developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values)

Reply:
<img width="449" alt="Screenshot 2024-02-13 at 10 51 43 AM" src="https://github.com/mbta/skate/assets/912020/8fc585ad-3cde-41d0-8d22-ab1647b95961">

Nailed it

Reply:
Okay but here we go... doing it correctly! https://github.com/mbta/skate/pull/2383/commits/e40fec99962d5201d3358ef704403ebc094acb1c